firmware/test/consoleFnTests.cpp

```diff
@@ -0,0 +1,338 @@
+#define CATCH_CONFIG_MAIN
```

Comment:
I suggest naming this function with `consoleFunctionsTests.cpp` to match the exact name of the file that you're testing.